src/core/Syntax.mli

```diff
@@ -26,6 +26,11 @@ module Make : functor (Symbol : Symbol.S) -> sig
   (** Vertically print an iterated dependent product type as if it were a sequent, for display of goals. *)
   val pp_sequent : lbl:string option -> (Ident.t * tp) list -> tp Pp.printer
 
+  (** Vertically print an iterated dependent product type as if it were a sequent, for display of goals.
+      This variant will also print out a partially constructed terms, as well as display if the boundary
+      conditions are met. *)
+  val pp_partial_sequent : bdry_sat:bool -> (Ident.t * tp) list -> (t * tp) Pp.printer
```

Comment:
what about using a variant instead of labeled boolean? i don’t feel strongly though